src/main/resources/export-build.qute.gradle

```diff
@@ -43,9 +43,12 @@ dependencies {
 application {
 	mainClass = '{fullClassName}'
 	applicationDefaultJvmArgs = [{jvmArgs}]
+{#if enablePreview eq 'true'}
+	applicationDefaultJvmArgs += ['--enable-preview']
+{/if}
 }
 
-{#if enableJavaPreview eq 'true'}
+{#if enablePreview eq 'true' and language eq 'java'}
```

Comment:
I *think* you also need to set specific release/source flag  - you can see how we do it in jbang itself at

https://github.com/maxandersen/jbang/blob/main/src/main/java/dev/jbang/source/buildsteps/CompileBuildStep.java#L74

Reply:
@maxandersen Just as an aside, we should probably use `--release` instead of `-source` for JDKs >= 9. I'll open an issue for that

Reply:
//PREVIEW now adds  the following additional compiler options for Java.

```
options.compilerArgs += ['--release', '{javaVersion}', '--enable-preview', '-Xlint:preview']
```

I see that others recommend adding -Xlink:preview to provide a  reminder to the user of the  specific preview features that are in use.

```
App/src/main/java/App.java:15: warning: [preview] unnamed classes are a preview feature and may be removed in a future release.
void main(String[] args) {
^
1 warning
```